### PR TITLE
Create Project.toml, drop REQUIRE

### PR DESCRIPTION
**Pull request by dm3**

### CODE REVIEW

**Comment by morris25** [comment]
You can probably just drop the failing 0.6 and 0.7 jobs at this point

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name = "CodecLz4"
+uuid = "5ba52731-8f18-5e0d-9241-30f10d1ec561"
+
```
Comment (morris25) [inline Project.toml]:
```suggestion
version = "0.2.0"
```

**Comment by morris25** [comment]
It would also be nice if you could update the docs deployment with a Project file.

File: .appveyor.yml [hunk 2]
```diff
@@ -1,7 +1,5 @@
 environment:
   matrix:
-  - julia_version: 0.6
-  - julia_version: 0.7
   - julia_version: 1
```
Comment (omus) [inline .appveyor.yml]: Would be nice:
```suggestion
  - julia_version: 1
  - julia_version: 1.1
```

**Comment by morris25** [comment]
@JuliaRegistrator register()

**Pull request merged**